libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -566,26 +558,24 @@ impl NodeAPI for Greenlight {
         Ok(())
     }
 
-    async fn sweep(
-        &self,
-        to_address: String,
-        fee_rate_sats_per_vbyte: u64,
-    ) -> Result<WithdrawResponse> {
-        let mut client = self.get_client().await?;
+    async fn sweep(&self, to_address: String, fee_rate_sats_per_vbyte: u64) -> Result<Vec<u8>> {
```

Comment:
Let's make `fee_rate_sats_per_vbyte` a `u32`.

Reply:
Done. Had to do it all the way through to the udl file.